SSA/Projects/InstCombine/LLVM/EDSL.lean

```diff
@@ -161,6 +161,11 @@ def mkExpr (Γ : Ctxt (MetaLLVM φ).Ty) (opStx : MLIR.AST.Op φ) :
         | "llvm.icmp.sge" => pure <| Sum.inr LLVM.IntPredicate.sge
         | "llvm.icmp.slt" => pure <| Sum.inr LLVM.IntPredicate.slt
         | "llvm.icmp.sle" => pure <| Sum.inr LLVM.IntPredicate.sle
+        -- | "llvm.icmp" => do
+        --   let attr? := opStx.attrs.getAttr "predicate"
+        --   match attr? with
+        --     | .none => throw <| .generic s!"What?"
+        --     | .some y => throw <| .generic s!"Predicate flag: {MLIR.AST.docAttrVal y}."
```

Comment:
You probably want to drop this in this PR and work on it in a different PR, no?

Reply:
Oops, yes